e2e/testcafe-devextreme/eslint.config.mjs

```diff
@@ -0,0 +1,256 @@
+/* eslint-disable spellcheck/spell-checker */
+import noOnlyTests from 'eslint-plugin-no-only-tests';
+import i18N from 'eslint-plugin-i18n';
+import babelParser from '@babel/eslint-parser';
+import tsParser from '@typescript-eslint/parser';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+import { FlatCompat } from '@eslint/eslintrc';
+import stylistic from '@stylistic/eslint-plugin';
+import importPlugin from 'eslint-plugin-import';
+import { changeRulesToStylistic } from 'eslint-migration-utils';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+});
+
+export default [
+  {
+    ignores: [
+      'node_modules/**',
+    ],
+  },
+  ...compat.extends('devextreme/spell-check'),
+  ...compat.extends('devextreme/testcafe'),
+  {
+    plugins: {
+      'no-only-tests': noOnlyTests,
+      i18n: i18N,
+    },
+    settings: {
+      'import/resolver': {
+        node: {
+          extensions: ['.js', '.jsx', '.ts', '.tsx', '.d.ts'],
+        },
+      },
+    },
+  },
+  {
+    ...js.configs.recommended,
+    ...importPlugin.flatConfigs.recommended,
```

Comment:
I think that rules from these plugins won't apply, because we override rules object [here](https://github.com/DevExpress/DevExtreme/pull/30845/files#diff-f0cd150c7c97ba706fe50f859d8e1907d016645692afdbb77b5792d83a7c3548R62).

Reply:
It's better to extend these plugins with something like this:

```js
...compat.extends('eslint/recommended', 'plugin:import/recommended').map(config => ({
    ...config,
    files: ['**/*.js']
  })),
```

Reply:
done